sdcm/cluster_k8s/mini_k8s.py

```diff
@@ -610,10 +610,12 @@ def on_deploy_completed(self):  # pylint: disable=too-many-branches
         if self.params.get('k8s_enable_tls'):
             images_to_cache.extend(self.ingress_controller_images)
 
-        try:
-            images_to_cache.append(self.get_operator_image())
-        except ValueError as exc:
-            LOGGER.warning("scylla-operator image won't be cached. Error: %s", str(exc))
+        # TODO: enable caching of the 'scylla-operator' image when the following bug gets fixed:
+        #       https://github.com/scylladb/scylla-operator/issues/1448
```

Comment:
https://github.com/scylladb/scylla-operator/issues/1448 was closed

i.e. no one gonna fix it, and it assumes it's kind bug, maybe we should report it in the right place ?

Reply:
We can link the bugreport for the `kind` tool later in the operator's bugreport.
So, reference will be clear.
It means, I propose to merge it to unblock all our local K8S jobs.

Reply:
o.k.